.gitignore

```diff
@@ -37,6 +37,8 @@ cypress.env.json
 # certificates
 ./certificates/localhost.crt
 ./certificates/localhost.key
+certificates/localhost-key.pem
+certificates/localhost.pem
```

Comment:
Remove this if it is not needed.

Reply:
This was added to make it easier to run https in dev mode, and prevent the locally generated certificates from being uploaded to the repo